src/Libraries/Microsoft.Extensions.AI.Abstractions/UsageDetails.cs

```diff
@@ -19,8 +21,23 @@ public class UsageDetails
     /// <summary>Gets or sets the total number of tokens used to produce the response.</summary>
     public int? TotalTokenCount { get; set; }
 
-    /// <summary>Gets or sets additional properties for the usage details.</summary>
-    public AdditionalPropertiesDictionary? AdditionalProperties { get; set; }
```

Comment:
We don't strictly have to remove `AdditionalProperties`, but I think it's a lot clearer if we do.

I'm not sure why someone would need to store usage data on `AdditionalProperties` now they have `AdditionalValues` (or could use `ChatCompletion.AdditionalProperties` if it's not usage data), and if they did so we couldn't aggregate it.